site/docs/tutorials/abstract-getting-started.mdx

```diff
@@ -0,0 +1,132 @@
+---
+name: Getting started with Abstract
+route: /tutorials/abstract-getting-started
+menu: Tutorials
+---
+
+import LargeParagraph from "../../src/components/LargeParagraph";
+import Image from "../../src/components/Image";
+import Link from "../../src/components/Link";
+
+# Getting started with Abstract
+
+<LargeParagraph>
+    If you are setting up a new design project or you are new to Abstract, this tutorial will help you to get started.
+</LargeParagraph>
+
+## Getting started
+
+### What is Abstract?
+Abstract is a design collaboration platform. It allows storing Sketch files and acts as a version control for them. Abstract allows you to make changes to the Sketch files without losing the original file. If needed, you can always return back to the original design.
+
+Abstract also allows you to collaborate with other Helsinki designers and to share your designs with developers. You can also easily browse other Helsinki design projects for inspiration and examples.
+
+### How to download and install Abstract?
+<p>Abstract has both a desktop and a web app. If you are just viewing design files, you can use the web app. If you are a designer, you need to download the desktop app to edit Sketch files. You can download the app from the <Link href="https://app.abstract.com/download" external>Abstract website download page</Link>. Note that you will need a Mac computer to use the Abstract desktop app.</p>
+
+### Which Abstract role do you need?
+Abstract has two (2) levels of access rights; **Viewer** and **Contributor**. 
+
+Users with the **Viewer** role can browse projects and their files and view detailed design specifications in the Inspect mode. Users with the **Contributor** role have the same features as the Viewer role does but also can create and edit Abstract projects. 
+
+As a rule of thumb, if you need to create or edit design files in Abstract, you need the Contributor role.
+
+<p>If you already are part of the City of Helsinki organisation but missing the Contributor role, ask the city of Helsinki representative of your project to fill out the <Link href="https://forms.office.com/r/ebx0YFYspB" external>Abstract invite form</Link> for you (requires Helsinki AD credentials).</p>
+
+### How to get invited to the City of Helsinki Abstract organisation?
+All City of Helsinki Abstract projects are under an organisation. To access these projects, you need an invite.
+
+<p>First, you need to register to Abstract. You can do that on the <Link href="https://app.abstract.com/signup" external>Abstract sign up page</Link>. You may use any email address you like, for example the email of your own organisation.</p>
```

Comment:
You may use any email address you like, for example, the email of your organization.